src/pages/home/sidebar/SidebarScreen/index.js

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
-import useWindowDimensions from '@hooks/useWindowDimensions';
 import FreezeWrapper from '@libs/Navigation/FreezeWrapper';
 import BaseSidebarScreen from './BaseSidebarScreen';
 import sidebarPropTypes from './sidebarPropTypes';
 
 function SidebarScreen(props) {
-    const {isSmallScreenWidth} = useWindowDimensions();
-
     return (
-        <FreezeWrapper keepVisible={!isSmallScreenWidth}>
+        <FreezeWrapper>
```

Comment:
Why did you remove this? Is the sidebar still clickable if you e.g. navigate to different reports twice? I think it was handling a case where you have the sidebar visible all the time and navigate deeper in the main stack, but maybe the navigation structure changed in a way that it is not necessary right now?

Reply:
FYI I tested this fix locally and the list is still fully clickable on the web, and causes no issues on mobile. It seems plausible that the nav structure changed and this is no loner a case